pkgs/servers/atlassian/jira.nix

```diff
@@ -30,6 +36,9 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     cp -rva . $out
+    for f in $out/bin/*.sh; do
+      wrapProgram $f --prefix PATH : ${stdenv.lib.makeBinPath [ gawk ]}
+    done
```

Comment:
The wrapping seems to be interfering with starting jira in my test deployment, both with openjdk and oraclejdk. How did you test this?

Reply:
Thank you for the review. 
I'm sorry to have wasted your time. 
I did a mistake.
I'll be back with a good job.